MovieAppPetProject/MovieAppPetProject/HomeScreen/NewHomeViewController.swift

```diff
@@ -45,7 +45,7 @@ class NewHomeViewController: UIViewController, UITableViewDelegate, UITableViewD
             guard let cell = tableView.dequeueReusableCell(withIdentifier: Constants.Identifiers.homeScreenTableCell, for: indexPath) as? NewTableViewCell else {
                 return UITableViewCell()
             }
-            cell.configure(with: nowPlayingViewModel.nowPlayingMovies ?? [], sectionTitle: " Now Playing")
+            cell.configure(with: nowPlayingViewModel.nowPlayingMovies ?? [], sectionTitle: "  Now Playing")
```

Comment:
"  Now Playing" Why the extra space? 
Same with all the others?

Reply:
Its for styling in the label, without those spaces in front it just feels a bit squashed into the left side with the rounded corners

Reply:
makes sense, thank you. will try keep the same effect but use padding then 👍

Reply:
chatted offline🙏🏼